BoxContentSDKSampleApp/shareExtension/ShareViewController.m

```diff
@@ -61,6 +65,14 @@ - (void)didSelectPost {
     NSExtensionItem *item = self.extensionContext.inputItems.firstObject;
     NSItemProvider *itemProvider = item.attachments.firstObject;
 
+    if (![itemProvider hasItemConformingToTypeIdentifier:typeIdentifier]) {
+        typeIdentifier = (__bridge_transfer NSString *)kUTTypeMovie;
+
+        if (![itemProvider hasItemConformingToTypeIdentifier:typeIdentifier]) {
+            typeIdentifier = (__bridge_transfer NSString *)kUTTypeZipArchive;
```

Comment:
Not guaranteed though, right? Should it be `kUTTypeData` or `kUTTypeItem` or something more generic?